vulnfeeds/vulns/vulns.go

```diff
@@ -163,10 +164,11 @@ func (affected *Affected) AttachExtractedVersionInfo(version cves.VersionInfo) {
 
 // PackageInfo is an intermediate struct to ease generating Vulnerability structs.
 type PackageInfo struct {
-	PkgName     string           `json:"pkg_name,omitempty" yaml:"pkg_name,omitempty"`
-	Ecosystem   string           `json:"ecosystem,omitempty" yaml:"ecosystem,omitempty"`
-	PURL        string           `json:"purl,omitempty" yaml:"purl,omitempty"`
-	VersionInfo cves.VersionInfo `json:"fixed_version,omitempty" yaml:"fixed_version,omitempty"`
+	PkgName           string            `json:"pkg_name,omitempty" yaml:"pkg_name,omitempty"`
+	Ecosystem         string            `json:"ecosystem,omitempty" yaml:"ecosystem,omitempty"`
+	PURL              string            `json:"purl,omitempty" yaml:"purl,omitempty"`
+	VersionInfo       cves.VersionInfo  `json:"fixed_version,omitempty" yaml:"fixed_version,omitempty"`
+	EcosystemSpecific map[string]string `json:"ecosystem_specific,omitempty" yaml:"ecosystem_specific,omitempty"`
```

Comment:
Is there a reason this ecosystem specific field is duplicated here? (already added one in Affected, and the schema only has one in affected, not package)